pkg/controller/disruption/disruption_test.go

```diff
@@ -30,10 +30,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/apimachinery/pkg/util/uuid"
+	"k8s.io/client-go/informers"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/kubernetes/pkg/api"
-	informers "k8s.io/kubernetes/pkg/client/informers/informers_generated/externalversions"
+	_ "k8s.io/kubernetes/pkg/api/install"
```

Comment:
I think instead we should migrate "api.Registry.GroupOrDie(v1.GroupName).GroupVersion.String()" to "testapi" in /pkg/controller/disruption/disruption_test.go.